scico/linop/xray/_xray.py

```diff
@@ -193,3 +194,116 @@ def _calc_weights(x0, dx, nx, angle, y0):
     weights = jnp.minimum(distance_to_next, width) / width
 
     return inds, weights
+
+
+class Parallel3dProjector:
```

Comment:
Make this
```python
class Parallel3dProjector:  # pragma: no cover
```
to avoid coverage complaints based on CPU-only CI tests.

Reply:
Actually, it's only the Astra 3D projector that is GPU-only, so consider this comment withdrawn.